products/pluto/metadata.yml

```diff
@@ -1,675 +1,1705 @@
 name: primary_land_use_tax_lot_output_pluto
 display_name: Primary Land Use Tax Lot Output (PLUTO)
-summary: "Extensive land use and geographic data at the tax lot level in comma\u2013\
-  separated values (CSV) file format. The PLUTO files contain more than seventy fields\
-  \ derived from data maintained by city agencies.\r\n\r\nAll previously released\
-  \ versions of this data are available at <a href=\"https://www1.nyc.gov/site/planning/data-maps/open-data/bytes-archive.page?sorts[year]=0\"\
-  >BYTES of the BIG APPLE- Archive</a>"
-description: "Extensive land use and geographic data at the tax lot level in comma\u2013\
-  separated values (CSV) file format. The PLUTO files contain more than seventy fields\
-  \ derived from data maintained by city agencies.\r\n\r\nAll previously released\
-  \ versions of this data are available at <a href=\"https://www1.nyc.gov/site/planning/data-maps/open-data/bytes-archive.page?sorts[year]=0\"\
-  >BYTES of the BIG APPLE- Archive</a>"
+summary: ""
+description: >-
+  Extensive land use and geographic data at the tax lot level in comma–separated
+  values (CSV) file format. The PLUTO files contain more than seventy fields derived
+  from data maintained by city agencies.
+
+  All previously released versions of this data are available at
+  <a href=\"https://www1.nyc.gov/site/planning/data-maps/open-data/bytes-archive.page?sorts[year]=0\">BYTES of the BIG APPLE- Archive</a>
 tags: [primary land use tax lot output, pluto, parcels, tax lot, tax block, zoning, land use, department of city planning, dcp, new york city, nyc]
-each_row_is_a: <FILL_ME_IN>
+each_row_is_a: Tax Lot
+
 destinations:
-- id: socrata_prod
+- id: socrata_prod_water_incl
   type: socrata
   four_four: 64uk-42ks
-  attachments: []
+  attachments:
```

Comment:
I noticed `attachments` tag is here and on the line 43. Can they have different attachments?

Reply:
I think they can, especially when we expand the destinations beyond just `socrata`